diagnostic.md

```diff
@@ -73,13 +109,27 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "PATCH" "http://www.example.com/donuts" \
+  --header "Content-Type: application/json" \
+  --data "{
+    \"donuts\": {
+      \"name\": \"French Cruller\",
+      \"price\": \"$0.99\"
+    }
+  }"
```

Comment:
This is a create request so the action would be `POST` not `PATCH`. Otherwise Correct.